### PR TITLE
Documentation iteration

### PR DESCRIPTION
**Pull request by marians**
This is an attempt to make the documentation easier to read and digest.

- Moved the docs for `template nodepool` into a separate page.
- Reduced info depth in the main README
- Use wording that reflects user goals

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,24 @@
-# kubectl-gs
+# `kubectl-gs` - kubectl Plug-In for Giant Swarm custom resources
+
+Plug-in for `kubectl` to create manifests for creating/updating custom resources for:
+
+- Clusters - see [`docs/template-cluster-cr.md`](https://github.com/giantswarm/kubectl-gs/blob/master/docs/template-cluster-cr.md) for details
+- Node pools
```
Comment (corest) [inline README.md]:
```suggestion
- Node pools - see ['docs/template-nodepool-cr.md '](https://github.com/giantswarm/kubectl-gs/blob/master/docs/template-nodepool-cr.md) for details
```

Comment (marians) [reply README.md]: Thanks!

File: docs/template-cluster-cr.md [hunk 2]
```diff
@@ -1,53 +1,47 @@
...
 
 
-#### Example
+## Example
 
-Running command
+Example command:
 
 ```
 gs template cluster \
```
Comment (corest) [inline docs/template-cluster-cr.md]:
```suggestion
kubectl gs template cluster \
```

File: docs/template-nodepool-cr.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
+# Creating a node pool based on custom resources using kubectl-gs
+
+Node pools are groups of worker nodes sharing common configuration. In terms of custom resources they consist of
+
+- `MachineDeployment` (API version `cluster.x-k8s.io/v1alpha2`)
+- `AWSMachineDeployment` (API version `infrastructure.giantswarm.io/v1alpha2`)
+
+## Usage
+
+To create the manifests for a new node poo, use this command:
```
Comment (corest) [inline docs/template-nodepool-cr.md]:
```suggestion
To create the manifests for a new node pool, use this command:
```

Comment (corest) [reply docs/template-nodepool-cr.md]: `node poo` sounds funny :D

**Comment by marians** [comment]
@pipo02mix Shall I join anyway?

**Comment by pipo02mix** [comment]
yes sure, my docs are into apiextensions and official docs, I will share today my work

**Pull request merged**